README.md

```diff
@@ -77,6 +83,10 @@ may be out of date, the source of truth is whether the nearest anchor to a norma
    not to meet requirements, but if you cannot find *any* reading system that processes the test as expected, that may
    indicate an implementation mistake in the test. Fix as necessary.
 
+1. Run the `.epub` through the online [EPUB Validator](http://validator.idpf.org/) or the
+   [EPUBCheck](https://www.w3.org/publishing/epubcheck/) command-line tool to ensure you didn't make any silly mistakes. Fix
+   if you did.
+
 1. Create a pull request for your test change, including both the uncompressed folder and the compressed `.epub` file. Please
    ensure the PR's description clearly indicates which statement is being tested. Await review.
 
```

Comment:
The URL-s to be added to the data-tests attributes are now different. They should be of the form `#{testid}`